airflow/models/taskinstance.py

```diff
@@ -761,6 +767,7 @@ def get_triggering_events() -> dict[str, list[DatasetEvent | DatasetEventPydanti
         "dag_run": dag_run,
         "data_interval_end": timezone.coerce_datetime(data_interval.end),
         "data_interval_start": timezone.coerce_datetime(data_interval.start),
+        "dataset_events": DatasetEventAccessors(),
```

Comment:
I was searching our codebase but could not find a good documentation about the content of `context`as being published. Would it now (with this PR) be time to document this for users?

I first thought of docs/apache-airflow/templates-ref.rst but this is only for templating, content-wise it overlaps a lot with the content in context.

Reply:
There is `templates-ref.rst` but I totally forgot to update it. I think I'll also write a pre-commit hook to make sure it's updated properly for this kind of things...

Do you think there's a way to improve the page's visibility?

Reply:
No, I found the `template-ref.rst` but did not qualify this in my comment - mainly because it is targeted for the usage of Jinja in templating - it has 80% overlap with the context but also a bit different details I think. But anyway, having docs there is also good.